evm/src/main/java/org/hyperledger/besu/evm/frame/MessageFrame.java

```diff
@@ -1653,6 +1674,7 @@ private void validate() {
         checkState(worldUpdater != null, "Missing message frame world updater");
         checkState(originator != null, "Missing message frame originator");
         checkState(gasPrice != null, "Missing message frame getGasRemaining price");
+        checkState(blobGasPrice != null, "Missing message frame blob gas price");
```

Comment:
This will break downstream users.  Make the default zero or use `Optional<Wei>` and default it to empty.